packages/replayio/src/commands/record.ts

```diff
@@ -52,7 +55,12 @@ async function record(url: string = "about:blank") {
     logPromise(promise, {
       messages: {
         pending: "Uploading crash data...",
-        success: "Crash data uploaded successfully",
+        success: () => {
+          if (uploadableCrashes.some(recording => recording.uploadStatus === "failed")) {
+            return "Some of the crash data couldn't be uploaded";
+          }
+          return "Crash data uploaded successfully";
+        },
```

Comment:
I kind of love how useful the little `logPromise` helper is proving to be 😁

Reply:
Useful stuff :)